opencog/nlp/chatbot/README

```diff
@@ -135,19 +83,18 @@ prefacing comments with cog:, cogita: or cogita-bot:
 
 Architecture:
 -------------
-The current architecture makes use of several servers interconnected
-by TCP/IP sockets to perform NLP processing.  The infrastructure is
-currently very minimalistic, and is just barely enough to get the job
-done. All modifications require tampering the source code. The pipeline
-is as follows:
+The current architecture makes use of several servers interconnected by 
+TCP/IP sockets to perform NLP processing.  
+
+The pipeline is as follows:
 
-1) IRC I/O, performed by cogita, which acts as a intermediary between
+1) IRC I/O, performed by cogita, acts as an intermediary between
    IRC and OpenCog.  It listens for input on an IRC channel, and
-   forwards the resulting plain-text to opencog. This is done by
-   issuing one simple scheme expression to opencog, via the opencog
+   forwards the resulting plain-text to OpenCog. This is done by
+   issuing one simple scheme expression to OpenCog, via the OpenCog
    command-line interface/scheme shell on port 17004. The expression
-   is ''(say-id-english usernick text)'', where the ''usernick'' is
-   the user's IRC nick, and ''text'' is what the user entered.  The
+   is ''(process_query user query)'', where the ''user'' is
+   the user's IRC nick, and ''query'' is what the user entered.  The
```

Comment:
well, `text` is better than `query` since the chatbot can accept statements.  You can tell it facts.